modules/visual_script/visual_script.h

```diff
@@ -270,6 +269,7 @@ class VisualScript : public Script {
 	static void _bind_methods();
 
 public:
+	void _set_variable_info(const StringName &p_name, const Dictionary &p_info); // TODO: Review, I think set_variable_info can be private, and _set_variable_info can be public (switch private with the public)
```

Comment:
I can't access to this function with the new signal system because was private.

I have a long explanation for this if you want, it took me hours to me to figure out what is going on.

Resume: This private function calls the public one. And the public is never used (except from the private one). I don't know, but seems to be upside down.

I test this and is working.